rng/src/vhu_rng.rs

```diff
@@ -211,6 +211,30 @@ impl VuRngBackend {
     }
 }
 
+#[cfg(test)]
```

Comment:
Interesting to add more functions based on cfg-test right here. I am not sure what's the standard way of doing it. I have only seen test specific code only in the test module until now. Anyway, this may be just fine.

@andreeaflorescu @lauralt any opinions ?

Reply:
Actually, you can move this impl block in the tests module and it will work just fine. I2C crate already does something like this in main.rs

Reply:
You can have it either in `mod tests` or in here. The disadvantage of having it here is that it will count towards the coverage score, so you will get a test coverage that is not accurate (i.e. it will show like your coverage is larger that it actually is). The only reason why we are sometimes placing such code outside of the mod tests is when we want to reuse the code blocks in another module. But even that can be achieved by making mod tests a public module and import the functionality in the target module.